scanapi/settings.py

```diff
@@ -14,17 +18,20 @@ def __init__(self):
         super(dict, self).__init__()
 
     def save_config_file_preferences(self, config_path=None):
-        if not config_path and not self.has_default_config_file:
-            return
-
-        if not config_path:
-            user_config = load_config_file(DEFAULT_CONFIG_PATH)
-            self["config_path"] = DEFAULT_CONFIG_PATH
-            self.update(**user_config)
+        if not any(
+            [config_path, self.has_local_config_file, self.has_global_config_file]
+        ):
             return
```

Comment:
Do we really need these check? The next couple of lines will perform if-elifs, and if all options are false, then it will already return None, like what is being done over here.
```suggestion
```

Reply:
Yea I agree I'll change that.